CaPPMS/Shared/ProjectEntryAuthorizedDropdown.razor

```diff
@@ -304,11 +337,60 @@
         NavigationManager.NavigateTo($"{ProjectManager.FileManager.DownloadPath}{location}", true);
 
     }
+    private async Task ApproveIdea2(bool confirmed)
+    {
+        if (!confirmed)
+        {
+            return;
+        }
+        var gitHubOrganization = "umgc";
+        var formattedRepoName = projectInformation.ProjectTitle.Replace(' ', '-');
+        try
+        {
+            this.gitHubState = "none";
+            await GitHub.doAllTasks(gitHubOrganization, formattedRepoName, projectInformation.ProjectDescription).ConfigureAwait(false);
+            this.gitHubState = "block";
+        }
+        catch (Exception ex)
+        {
+            Console.WriteLine("Error Message - {0}", ex.Message);
+            this.gitHubState = "block";
+            await Task.Delay(2000).ConfigureAwait(false);
```

Comment:
This dealy shouldn't be needed here I think. What are you using it for?

Reply:
So this is for a better user experience. Without the wait, the attempt to create the repository isn't visible to the user. It will output "Trying to create repository" and wait for 2 seconds to give user more feedback. Then fail to a new displaying them a new error message. Totally optional, just thought a better overall user experience to have more insight on what's going on in the background.